modules/gdscript/language_server/gdscript_language_server.cpp

```diff
@@ -76,9 +78,10 @@ void GDScriptLanguageServer::thread_main(void *p_userdata) {
 }
 
 void GDScriptLanguageServer::start() {
+	host = String(_EDITOR_GET("network/language_server/remote_host"));
 	port = (int)_EDITOR_GET("network/language_server/remote_port");
 	use_thread = (bool)_EDITOR_GET("network/language_server/use_thread");
-	if (protocol.start(port, IPAddress("127.0.0.1")) == OK) {
+	if (protocol.start(port, IP_Address(host)) == OK) {
```

Comment:
It seems you are using an outdated version, this was renamed to `IPAddress` (without underscore) a while ago.

```suggestion
	if (protocol.start(port, IPAddress(host)) == OK) {
```